assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -14,6 +14,7 @@ spec:
   selector:
     matchLabels:
       app: kube-apiserver
+  minReadySeconds: 30
```

Comment:
Since this deployment utilizes a `readinessProbe`, we should lower `minReadySeconds` to `15`.

Reply:
updated